packages/contracts/foundry_test/WhaleSniper.t.sol

```diff
@@ -132,8 +132,10 @@ contract WhaleSniperPOCTest is eBTCBaseFixture {
                 cdpManager.MAX_REWARD_SPLIT()) / cdpManager.stakingRewardSplit();
 
             // hack manipulation to sync global index in attacker's benefit
-            uint256 _oldIdx = _newIndex - _requiredDeltaIdxTriggeRM - 1234567890;
+            uint256 _oldIdx = _newIndex - _requiredDeltaIdxTriggeRM;
```

Comment:
curious on reasoning for this change, or the original setup

Reply:
This can overflow in some scenarios